docs/user-guide/getting-started.md

```diff
@@ -171,28 +175,193 @@ Usage: measure-engine [options]
   * -c, --context-id
       FHIR resource ID for one or more patients to evaluate.
   * -d, --data-server
-      Path to JSON configuration data for the FHIR server connection that will
+      Path to JSON configuration data for the FHIR server connection that will 
       be used to retrieve data.
+    -f, --format
+      Output format of the report (JSON|TEXT*)
+      Default: TEXT
+      Possible Values: [TEXT, JSON]
     -h, --help
       Display this help
+  * -p, --measure-parameters
+      JSON File containing measure resource ids and optional parameters
     -m, --measure-server
-      Path to JSON configuration data for the FHIR server connection that will
+      Path to JSON configuration data for the FHIR server connection that will 
       be used to retrieve measure and library resources.
-    -p, --parameters
-      Parameter value(s) in format name:type:value where value can contain
-      additional parameterized elements separated by comma
-  * -r, --resource
-      FHIR Resource ID for the measure resource to be evaluated
     -t, --terminology-server
-      Path to JSON configuration data for the FHIR server connection that will
+      Path to JSON configuration data for the FHIR server connection that will 
       be used to retrieve terminology.
+```
+
+## JSON File for -p Argument
+
+The `-p` argument should be the path to a file containing a JSON object containing a list of measure resource ids and 
+any parameters that should be used when executing a particular measure.
+
+Outer Structure:
+```text
+{
+   "measureParameters" : [
```

Comment:
I like the sound of measureConfiguration.

No need to be disappointed. I can add the feature back and make it so you can pass in JSON or just specify a single measure on the command line.

Reply:
Pushed some changes this afternoon. Any thoughts on the parameter names? I am currently using "--measure-configurations" and "-e" (since "-m" and "-c" are claimed already). I'd be open to suggestions.

Reply:
Maybe change things around to reference JSON in some way? So perhaps "--json-measure-configurations" and "-j"?